src/screens/Welcome/WelcomeScreen.js

```diff
@@ -10,8 +10,12 @@ const civicLogoIcon = require('../../assets/images/civic-logo-white.png');
 const WelcomeScreen = props => {
   return (
     <View style={styles.container}>
-      <Image style={styles.civicLogo} source={civicLogoIcon} />
-      <WelcomeCarousel isSmallScreen={props.isSmallScreen} />
+      <View></View>
+      <View style={styles.mastheadContainer}>
+        <Image style={styles.civicLogo} source={civicLogoIcon} />
+        <WelcomeCarousel isSmallScreen={props.isSmallScreen} />
+      </View>
+      <View></View>
```

Comment:
And this one too

Reply:
@gmferland Hi Greg, those empty View components are there intentionally for flex spacing. I noticed you had mentioned in a previous issue that getting the layout to look proper on multiple phone screens was an issue for the Welcome screen.

By adding these empty View components within the parent container that is flexed, they will eat up the extra vertical space in a proportional manner and create a more consistent layout across devices. I grouped the logo and carousel together so that the spacing between them would remain fixed.

Not sure if there is a different component that I could use as a filler, I use View because it seemed most similar to a div.

Reply:
Ah, thanks for clarifying that. I've done similar things with divs before as well. I was surprised to see no styling on those Views since in my experience, an empty div would be ignored by the flexbox layout unless it had some sort of width or content. Maybe the react native pseudo-flexbox is different. I'm glad you were able to solve some of the layout issues!